generated/java/proto-google-iam-v1/build.gradle

```diff
@@ -22,7 +22,7 @@ repositories {
 dependencies {
   compile "com.google.protobuf:protobuf-java:3.4.0"
   compile "com.google.api:api-common:1.3.0"
-  compile project(":proto-google-common-protos")
+  compile "com.google.api.grpc:proto-google-common-protos:1.1.0"
```

Comment:
This is incorrect, because @neozwu had to roll back the fix for inter-project dependencies

Reply:
note taken